src/modules/search.js

```diff
@@ -229,6 +231,87 @@ class Search {
         throw new Error('getSearchResults response data is malformed');
       });
   }
+
+  /**
+   * Retrieve voice search results from API
+   *
+   * @function getVoiceSearchResults
+   * @description Retrieve voice search results from Constructor.io API
+   * @param {string} query - Term to use to perform a voice search
+   * @param {object} [parameters] - Additional parameters to refine result set
+   * @param {number} [parameters.page] - The page number of the results(Can't be used together with offset)
```

Comment:
Could we add a space before the parentheses?

Reply:
Eyes of an Eagle